### PR TITLE
Bigscience176b

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

- Fixing minor issues / improvments on the previous PR

### CODE REVIEW

**Comment by younesbelkada** [comment]
This is a new update where I fix some minor integration issues, I have also managed to remove the for loop on the Softmax 527dd43f4161e889ceb110ba1e5adfd7f30be8c3. It appeared that I forgot to add the attention masks during my first tests, that is why the tests failed. Everything seems to work fine now, except for the Build PR doc test.

**Comment by younesbelkada** [comment]
Adding an updated version of the model definition
- Some operations were not consistent with the former definition 
- With a larger model on JZ the test pass with `rtol=1e-5, atol=1e-2` with `nn.LayerNorm` and `apex.normalization.FusedLayerNorm`
- Needs to check with the final model
